.github/workflows/phpstan.yml

```
@@ -0,0 +1,34 @@
+name: phpstan static analysis
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  phpstan:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate
```

```suggestion
      run: composer validate --strict
```

Will it work with strict?

---

took inspiration of the exising job for this settings

---

If the existing job already has this it doesn't make sense to add it again